generator_test.go

```diff
@@ -1101,6 +1101,7 @@ func TestDefaultHWAddrFunc(t *testing.T) {
 }
 
 func BenchmarkGenerator(b *testing.B) {
+	b.ReportAllocs()
```

Comment:
Personal opinion: it's cleaner to not have this directly in the benchmark and include `-benchmem` when running `go test`.  This way you **can't** only benchmark times without modifying the code.